templates/epic.xml.jinja2

```diff
@@ -97,131 +93,74 @@
      - and more...
   </documentation>
 
-{% if features is not defined or 'solenoid' in features %}
+{% if 'solenoid' in features %}
   <documentation level="5">
     ## Main magnet
   </documentation>
   <include ref="${DETECTOR_PATH}/compact/solenoid.xml"/>
 {% endif -%}
 
-{% if features is not defined or 'tracking' in features %}
+{% if 'tracking' in features %}
   <documentation level="10">
     ## Central tracking detectors
   </documentation>
   <include ref="${DETECTOR_PATH}/compact/tracking/definitions.xml"/>
 
-  {% if features is defined and features['tracking'] is not none -%}
-  {%   for tracker in features['tracking'] -%}
+  {% for tracker in features['tracking'] -%}
   <include ref="${DETECTOR_PATH}/compact/tracking/{{ tracker }}.xml"/>
-  {%   endfor -%}
-  {% else -%}
-  <include ref="${DETECTOR_PATH}/compact/tracking/vertex_barrel.xml"/>
-  <include ref="${DETECTOR_PATH}/compact/tracking/silicon_barrel.xml"/>
-  <include ref="${DETECTOR_PATH}/compact/tracking/mpgd_barrel.xml"/>
-  <include ref="${DETECTOR_PATH}/compact/tracking/mpgd_dirc.xml"/>
-  <include ref="${DETECTOR_PATH}/compact/tracking/silicon_disks.xml"/>
-  <include ref="${DETECTOR_PATH}/compact/tracking/support_service_assembly.xml"/>
-  <include ref="${DETECTOR_PATH}/compact/tracking/tof_barrel.xml"/>
-  <include ref="${DETECTOR_PATH}/compact/tracking/tof_endcap.xml"/>
-  {% endif -%}
+  {% endfor -%}
 {% endif -%}
 
-{% if features is not defined or 'pid' in features %}
+{% if 'pid' in features %}
   <documentation level="10">
     ## PID detectors
   </documentation>
-  {% if features is not defined or features['pid'] is none or 'dirc' in features['pid'] -%}
-  <include ref="${DETECTOR_PATH}/compact/dirc.xml"/>
-  {% endif -%}
-
-  {% if features is not defined or features['pid'] is none or 'drich' in features['pid'] %}
-  {%- if features is not defined or features['pid'] is none or features['pid']['drich'] is none %}
-  <include ref="${DETECTOR_PATH}/compact/drich.xml"/>
-  {% else %}
-  <include ref="{{ features['pid']['drich'] }}"/>
-  {% endif -%}
-  {% endif -%}
-
-  {% if features is not defined or features['pid'] is none or 'mrich' in features['pid'] %}
-  <include ref="${DETECTOR_PATH}/compact/mrich.xml"/>
-  {% elif 'pfrich' in features['pid'] -%}
-  <include ref="${DETECTOR_PATH}/compact/pfrich.xml"/>
-  {% endif -%}
+  {% for pid in features['pid'] -%}
+  <include ref="${DETECTOR_PATH}/compact/pid/{{ pid }}.xml"/>
+  {% endfor -%}
 {% endif -%}
 
-{% if features is not defined or 'ecal' in features %}
+{% if 'ecal' in features %}
   <documentation level="10">
     ## Central EM calorimetry
   </documentation>
-  {% if features is not defined or features['ecal'] is none or 'forward_homogeneous' in features['ecal'] %}
-  <include ref="${DETECTOR_PATH}/compact/ecal_forward_homogeneous.xml"/>
-  {% elif 'forward_scifi' in features['ecal'] -%}
-  <include ref="${DETECTOR_PATH}/compact/ecal_forward_scfi.xml"/>
```

Comment:
So which one it is, "scifi" or "scfi"? :)

Reply:
Historically? ScFi for imaging ecal. But we can't define what the forward ecal calls it...